linkcovery/services/data_service.py

```
@@ -77,93 +69,75 @@ def import_from_json(self, input_path: str | Path) -> None:
                     self.link_service.add_link(
                         url=url,
-                        description=description,
-                        tag=tag,
-                        is_read=is_read,
+                        description=link_data.get("description", asyncio_run(fetch_description(url=url))),
+                        tag=link_data.get("tag", ""),
+                        is_read=link_data.get("is_read", False),
                     )
                     added_count += 1
-
                 except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)